### PR TITLE
Ossp master update for gramine, logstash, threatplaybook, struts

### PR DESCRIPTION
**Pull request by sand-hya**
#343 #289 #292 #290

### CODE REVIEW

File: bes_theme/assets/data/OSSP-master.js [hunk 1]
```diff
@@ -1477,7 +1477,302 @@ const projectOfInterestObject = {
 			]
 
 		}
-
+		{
```
Comment (sudhirverma) [inline bes_theme/assets/data/OSSP-master.js:1480]: @sand-hya JSON format is not correct you need to provide comma after each object

File: bes_theme/assets/data/OSSP-master.js [hunk 2]
```diff
@@ -1477,7 +1477,302 @@ const projectOfInterestObject = {
...
+			"tags": [
+				"L&F",
+				"SD-AS",
+				"IND-ALL",
+				"COM-C",
+				
+			]
+		
+		}
+		{
```
Comment (sudhirverma) [inline bes_theme/assets/data/OSSP-master.js:1553]: same here as above comment

File: bes_theme/assets/data/OSSP-master.js [hunk 3]
```diff
@@ -1477,7 +1477,302 @@ const projectOfInterestObject = {
...
+				"IND-CyS",
+				"IND-ED",
+				"IND-RT",
+				"IND-TEL",
+				"COM-C",
+				
+			]
+		
+		}
+		{
```
Comment (sudhirverma) [inline bes_theme/assets/data/OSSP-master.js:1636]: same here as above comment

File: bes_theme/assets/data/OSSP-master.js [hunk 4]
```diff
@@ -1477,7 +1477,302 @@ const projectOfInterestObject = {
...
+				"IND-HLS",
+				"IND-NG",
+				"IND-RT",
+				"IND-TEL",
+				"COM-C",
+				
+			]
+		
+		}
+		{
```
Comment (sudhirverma) [inline bes_theme/assets/data/OSSP-master.js:1712]: same here as first comment

**Review by sudhirverma — CHANGES_REQUESTED** [review]

**Comment by sudhirverma** [comment]
Please tag the issue in description section

**Comment by asa1997** [comment]
@sand-hya Please address the comments.

**Review by asa1997 — APPROVED** [review]
@sand-hya LGTM

**Pull request merged**